mailpoet/tests/unit/Logging/LoggerFactoryTest.php

```diff
@@ -39,13 +39,13 @@ public function testItReturnsInstance() {
   public function testItAttachesProcessors() {
     $logger1 = $this->loggerFactory->getLogger('logger-with-processors', true);
     $processors = $logger1->getProcessors();
-    expect($processors)->notEmpty();
+    expect(count($processors))->greaterThan(1);
   }
 
   public function testItDoesNotAttachProcessors() {
     $logger1 = $this->loggerFactory->getLogger('logger-without-processors', false);
```

Comment:
Similar to a previous comment, I believe we should change the name of this logger and the name of this test to reflect that this logger actually includes one processor, but doesn't include "optional" processors. What do you think?